models/feedback.go

```
@@ -47,3 +46,36 @@ func (f *Feedback) Sanitize(cfg *config.Sanitize) {
 	f.Name = Sanitize(cfg, f.Name)
 	f.EmailAddress = Sanitize(cfg, f.EmailAddress)
 }
+
+// IsSiteDomainURL is true when urlString is a URL and its host ends with `.`+siteDomain (when siteDomain is blank, or uses config.SiteDomain)
+func IsSiteDomainURL(urlString, siteDomain string) bool {
+	if urlString == "" {
+		return false
+	}
+	urlString = NormaliseURL(urlString)
+	urlObject, err := url.ParseRequestURI(urlString)
+	if err != nil {
+		return false
+	}
+	if siteDomain == "" {
+		if cfg == nil {
+			if cfg, err = config.Get(); err != nil {
+				return false
+			}
```

Not sure I like this. We're masking an error here. We could at least be logging it.

---

Done